.github/CODEOWNERS

```diff
@@ -0,0 +1,295 @@
+/openstack/lyra                                        @databus23 @ArtieReus
+/openstack/maia                                        @notque @artherd42 @viennaa @richardtief @Kuckkuck @dhalimi1975
+/openstack/manila*                                     @Carthaca @chuan137 @galkindmitrii @kpawar-sap @sumitarora2786 @crenduchinta88
+/openstack/nannies                                     @thgrs @chuan137 @Carthaca @fwiesel @kpawar-sap
+/openstack/neutron                                     @notandy @fwiesel @sebageek @sven-rosenzweig @rhalina @swagner-de @mutax @BenjaminLudwigSAP @m-kratochvil @occamshatchet
+/openstack/nova                                        @joker-at-work @fwiesel @grandchild @leust
+/openstack/octavia                                     @notandy @BenjaminLudwigSAP @fwiesel @dusandordevicsap @velp @m-kratochvil
+/openstack/octobus                                     @viennaa
+/openstack/openstack-seeder                            @stefanhipfel @databus23 @fwiesel
+/openstack/openstack_performance                       @misamoylov
+/openstack/placement                                   @fwiesel @joker-at-work @bashar-alkhateeb @galkindmitrii
+/openstack/poller                                      @dhalimi1975 @corey-aloia
+/openstack/prometheus-openstack                        @viennaa @artherd42 @richardtief @Kuckkuck
+/openstack/pyroscope                                   @fwiesel
+/openstack/sap-seeds                                   @dhoeller @grandchild @frr2018 @joker-at-work @fwiesel
+/openstack/sre                                         @artherd42 @fwiesel @rajivmucheli
+/openstack/swift                                       @sumitarora2786 @crenduchinta88 @majewsky @SuperSandro2000 @VoigtS
+/openstack/swift-drive-autopilot                       @sumitarora2786 @crenduchinta88 @majewsky @SuperSandro2000 @VoigtS
+/openstack/swift-utils                                 @sumitarora2786 @crenduchinta88 @majewsky @SuperSandro2000 @VoigtS
+/openstack/tempest/barbican-tempest                    @misamoylov @fwiesel @rajivmucheli
+/openstack/tempest/cinder-tempest                      @misamoylov @fwiesel
+/openstack/tempest/designate-tempest                   @misamoylov @fwiesel
+/openstack/tempest/glance-tempest                      @misamoylov @fwiesel @rajivmucheli
+/openstack/tempest/ironic-tempest                      @misamoylov @fwiesel
+/openstack/tempest/keystone-tempest                    @misamoylov @fwiesel @rajivmucheli
+/openstack/tempest/manila-tempest                      @kpawar-sap @misamoylov @fwiesel
+/openstack/tempest/neutron-tempest                     @misamoylov @fwiesel
+/openstack/tempest/nova-tempest                        @misamoylov @fwiesel
+/openstack/tempest/octavia-tempest                     @misamoylov @fwiesel
+/openstack/tempest/tempest-base                        @misamoylov @fwiesel @stefanhipfel
```

Comment:
could you please add me for all tempest* paths?

Reply:
only `.../tempest*` or all `.../*tempest*`?

Reply:
eh, probably the latter.

Reply:
all /*tempest* please :)